app/components/post_draft/post_input/index.ts

```diff
@@ -59,4 +60,4 @@ const enhanced = withObservables([], ({database, channelId, rootId}: WithDatabas
     };
 });
 
-export default withDatabase(enhanced(PostInput));
```

Comment:
Protecting itself from changes on the parent. This is seen when PostDraft changes the postInputTop state. It only saves ~1ms in that case. Should verify if it is worth it in other situations.